File: Frontend/src/app/models/account.model.ts
import { ApartmentModel } from "./apartment.model";
import { OrderModel } from "./order.model";

export interface AccountModel{
    id: string;
    fullname: string;
    username: string;
    email: string;
    mobilePhone: string;
    role: string;
    creationDate: Date;
    lastlyUpdatedDate: Date;
}

File: Backend/DwellingAPI.Models/Models/AccountModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DwellingAPI.Shared.Models
{
    public class AccountModel
    {
        public string Id { get; set; } = string.Empty;
        public string Fullname { get; set; } = string.Empty;
        public string Username { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string MobilePhone { get; set; } = string.Empty;
        public string Role { get; set; } = string.Empty;

        public DateTime CreationDate { get; set; }
        public DateTime LastlyUpdatedDate { get; set; }
    }
}
